### ISSUE TITLE
"gradient experiments list" fails

### ISSUE THREAD

**Issue by josh-friedlander-kando** [Paperspace/gradient-cli#241]
Running `gradient experiments list` in the CLI fails with this stack trace:

```
Traceback (most recent call last):
  File "~/miniconda3/envs/gradient/bin/gradient", line 8, in <module>
    sys.exit(main.main())
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/main.py", line 7, in main
    _cli_entry_point()
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/cli/cli.py", line 17, in main
    super(GradientGroup, self).main(*args, **kwargs)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/cli/experiments.py", line 701, in list_experiments
    for experiments_str, next_iteration in res:
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/commands/common.py", line 73, in _generate_data_table
    **kwargs
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/commands/experiments.py", line 243, in _get_instances
    instances, meta_data = self.client.list(get_meta=True, **kwargs)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/api_sdk/clients/experiment_client.py", line 741, in list
    experiments = repository.list(project_id=project_id, limit=limit, offset=offset, get_meta=get_meta, tags=tags)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/api_sdk/repositories/common.py", line 97, in list
    instances = self._get_instances(response, **kwargs)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/api_sdk/repositories/common.py", line 108, in _get_instances
    objects = self._parse_objects(response.data, **kwargs)
  File "~/miniconda3/envs/gradient/lib/python3.6/site-packages/gradient/api_sdk/repositories/experiments.py", line 45, in _parse_objects
    experiment_dict.update(experiment_dict["templateHistory"]["params"])
KeyError: 'params'
```

Using version 0.6.0

**Comment by BartoszCki** [comment]
@josh-friedlander-kando 
Thanks for reporting this. We are investigating the issue.

**Comment by BartoszCki** [comment]
@josh-friedlander-kando 
Just published gradient 0.6.1 with a fix addressing your case.

Thanks again for reporting this. 
Stay safe and healthy!
